js/index.js

```diff
@@ -26,4 +26,68 @@ for(let i = 0; i < skills.length; i++) {
     skill.innerHTML = skills[i];
     skillList.appendChild(skill);
     skill.style.margin = '0 3em';
-}
\ No newline at end of file
+}
+
+// Creating the Submit Form
+const messageForm = document.querySelector('[name="leave_message"]');
+const leaveMessage = document.querySelector('#leave_message');
+
+messageForm.addEventListener('submit', submitMessage);
+
+function submitMessage(event) {
+    event.preventDefault(); // Prevent the form from submitting the default way
+    
+    const username = event.target.usersName.value;
+    const email = event.target.usersEmail.value;
+    const message = event.target.usersMessage.value;
+    
+    console.log(username, email, message);
+
+    const messageSection = document.querySelector('#messages');
+    const messageList = messageSection.querySelector('ul');
+    const newMessage = document.createElement('li'); //makes a new list item
+    messageList.appendChild(newMessage);
+
+    newMessage.innerHTML = `
+    <a href="mailto:${email}">${username}</a>
+    <span> ${message}</span>`;
+
+    // Remove button creation
+    const removeButton = document.createElement('button');
+    removeButton.innerHTML = "Remove";
+    removeButton.setAttribute("type", "button");
+    removeButton.setAttribute("class", "remove_button");
+
+    removeButton.addEventListener('click', function(){
+        const entry = removeButton.parentNode;
+        entry.removeChild(removeButton);
+        newMessage.remove();
+
+    // To hide the message section when empty
+        if(messageList.children.length === 0) {
+            messageSection.style.display = 'none';
+            leaveMessage.append(el);
+        }
+    });  
+   
+//   Creating the Edit button
+    const editButton = document.createElement('button');
+    editButton.innerHTML = "Edit";
+    editButton.setAttribute("type", "button");
+    editButton.setAttribute("class", "edit_button");
```

Comment:
Great job with the stretch goal of adding the edit button.

You're probably still working on this, but there doesn't seem to be any way to submit the text once it's edited.

Reply:
Yeah, I was not sure if I should add 'submit' button or just leave the 'edit' one